poprox-db/migrations/versions/2025_05_17_0754-1d20bc70fa6d_create_top_stories_table.py

```diff
@@ -0,0 +1,65 @@
+"""Create top_stories table
+
+Revision ID: 1d20bc70fa6d
+Revises: 45d10f860ff6
+Create Date: 2025-05-17 07:54:17.565202
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = '1d20bc70fa6d'
+down_revision: Union[str, None] = '45d10f860ff6'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "top_stories",
+        sa.Column("article_id", sa.UUID, nullable=False),
+        sa.Column("entity_id", sa.UUID, nullable=True),
```

Comment:
is the `entity_id` the topic for which this was listed? I felt like I understood this over in the concept, but I'm questioning myself now.

Reply:
Yeah, we create an entity for every topic, so I figured it might be better to link them to the topic than repeat the topic name here